src/NewPipelinePage.tsx

```diff
@@ -134,7 +134,7 @@ export default (): React.ReactElement => {
   }, [schemaObject, ajv]);
 
   const blockSettingsClickHandler = useCallback(
-    (e, node: NativeBlockModel) => {
+    (node: NativeBlockModel) => {
```

Comment:
the lamba receives an event and a node, if you remove the `e` arg, the event will be assigned to the `node` parameter

Reply:
the lambda arguments depends on how the lambda is called, which is what I changed to keep uniformity with the other callback handlers. I think the native event that resulted in this lambda being called is not relevant here anymore, thus we can drop the argument